### PR TITLE
PMM-11695 SQL injection.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**
PMM-11695

Link to the Feature Build: SUBMODULES-0

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

File: agent/runner/actions/mysql_explain_action.go [hunk 1]
```diff
@@ -164,7 +167,12 @@ func (a *mysqlExplainAction) explainDefault(ctx context.Context, tx *sql.Tx) ([]
 
 func (a *mysqlExplainAction) explainJSON(ctx context.Context, tx *sql.Tx) ([]byte, error) {
 	var b []byte
-	err := tx.QueryRowContext(ctx, fmt.Sprintf("EXPLAIN /* pmm-agent */ FORMAT=JSON %s", a.query)).Scan(&b)
+	inter := []any{}
+	for _, p := range a.params.Values {
+		inter = append(inter, p)
+	}
```
Comment (BupycHuk) [inline agent/runner/actions/mysql_explain_action.go:173]: can we convert it to some function?

File: api-tests/management/action/explain_test.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ func TestRunExplain(t *testing.T) {
 		Body: actions.StartMySQLExplainActionBody{
 			// PMMAgentID: "/agent_id/f235005b-9cca-4b73-bbbd-1251067c3138",
 			ServiceID: "/service_id/5a9a7aa6-7af4-47be-817c-6d88e955bff2",
-			Query:     "SELECT `t` . * FROM `test` . `key_value` `t`",
+			QueryID:   "3923dace316a86643fcf87cb45e0558a",
```
Comment (BupycHuk) [inline api-tests/management/action/explain_test.go:39]: let's make it a separate test.

File: api/managementpb/actions.proto [hunk 3]
```diff
@@ -47,8 +47,7 @@ message StartMySQLExplainActionRequest {
   string pmm_agent_id = 1;
   // Service ID for this Action. Required.
   string service_id = 2 [(validator.field) = {string_not_empty: true}];
-  // Query ID or query is required.
-  // Value of query.
+  // Deprecated: should not be used.
```
Comment (BupycHuk) [inline api/managementpb/actions.proto:50]: let's also add that it will be removed in 2.42

Comment (BupycHuk) [reply api/managementpb/actions.proto:50]: and in other places too.

**Review by BupycHuk — APPROVED** [review]

File: api/managementpb/actions.proto (hunk 3 above)
Comment (JiriCtvrtka) [reply api/managementpb/actions.proto:50]: Done.

File: agent/runner/actions/mysql_explain_action.go (hunk 1 above)
Comment (JiriCtvrtka) [reply agent/runner/actions/mysql_explain_action.go]: Sure. Done.

**Review by BupycHuk — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**